chapters/VK_NV_win32_keyed_mutex/keyed_mutex_submit.txt

```diff
@@ -5,14 +5,6 @@
 [open,refpage='VkWin32KeyedMutexAcquireReleaseInfoNV',desc='use Windows keyex mutex mechanism to synchronize work',type='structs']
 --
 
-When submitting work that operates on memory imported from a Direct3D 11
-resource to a queue, the keyed mutex mechanism may: be used in addition to
-Vulkan semaphores to synchronize the work.
-Keyed mutexes are a property of a properly created shareable Direct3D 11
-resource.
-They can: only be used if the imported resource was created with the
-etext:D3D11_RESOURCE_MISC_SHARED_KEYEDMUTEX flag.
-
```

Comment:
This seems to differ from the other changes, in that it's not a simple typo tweak. What was the rational for removing these sentences?

Reply:
I dropped the commit and filed an issue instead: https://github.com/KhronosGroup/Vulkan-Docs/issues/1435.